redhat-developer/developers.redhat.com
# New Search Revisions

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5172
* https://issues.jboss.org/browse/DEVELOPER-5903

### Verification Process
* Log in to Drupal
* Set the new fusion search page to published
* Check the page for functionality, behaviors, and results
